services/movie_session.py

```diff
@@ -52,10 +52,10 @@ def update_movie_session(
 
 
 def delete_movie_session_by_id(movie_session_id: int) -> None:
-    try:
-        movie_session = MovieSession.objects.get(id=movie_session_id)
+    movie_session = MovieSession.objects.filter(id=movie_session_id)
+    if movie_session.exists():
         movie_session.delete()
-    except MovieSession.DoesNotExist:
+    else:
         raise ValueError(
             f"MovieSession with id {movie_session_id} does not exist."
```

Comment:
No need to check if record exists when you use filter(). Filter will return an empty queryset and delete will not delete anything. This is why we often use filter instead of get